performance_regression_test.py

```diff
@@ -149,5 +149,33 @@ def test_simple_regression(self):
         except:
             pass
 
+    def test_read(self):
+        """
+        Test steps:
+
+        1. Run a write workload as a preparation
+        2. Run a read workload
+        """
+        base_cmd_w = ("cassandra-stress write no-warmup cl=QUORUM n=30000000 "
+                      "-schema 'replication(factor=3)' -port jmx=6868 "
+                      "-mode cql3 native -rate threads=500 -errors ignore "
+                      "-pop seq=1..30000000")
+        base_cmd_r = ("cassandra-stress read no-warmup cl=QUORUM duration=50m "
+                      "-schema 'replication(factor=3)' -port jmx=6868 "
+                      "-mode cql3 native -rate threads=500 -errors ignore "
+                      "-pop 'dist=gauss(1..30000000,15000000,1500000)' ")
+
+        # run a write workload
+        stress_queue = self.run_stress_thread(stress_cmd=base_cmd_w, stress_num=2)
+        results = self.get_stress_results(queue=stress_queue, stress_num=2)
+
+        stress_queue = self.run_stress_thread(stress_cmd=base_cmd_r, stress_num=2)
+        results = self.get_stress_results(queue=stress_queue, stress_num=2)
+
+        try:
+            self.display_results(results)
+        except:
+            pass
```

Comment:
Write stress is the preparation, we only care read result.
We can remove the first result assignment to avoid misunderstanding.